src/development/tools/devtools/memory.md

```diff
@@ -70,8 +81,8 @@ void myFunction() {
   
   Child? child = Child();
   
-  // The `child` object was allocated memory
-  // and is retained (exempt) from garbage collection
+  // The `child` object was allocated in memory.
+  // It is now retained from garbage collection
```

Comment:
Shams, I do not like 'exempt' here. It would apply if most object were garbage collected, but a few would be exempt. 
In reality, most objects are not garbage collected because of retaining paths, and just few are garbage collected, because they do not have retaining paths.

Does it make sense?